src/olympia/abuse/utils.py

```diff
@@ -285,6 +285,10 @@ def flag_for_human_review(self):
                 set(
                     self.target.versions(manager='unfiltered_for_relations')
                     .filter(version__in=reported_versions)
+                    .exclude(
```

Comment:
The point of this exclude is to prevent adding a NeedsHumanReview record for versions which already have an active one, right? Why do we need that additional filter now? Would it have made sense to already have it?

The changes below here seem to change specifically what type of activity log is produced, but this looks like it would change which versions get a NHR record. Am I reading that right?

Reply:
Yes, it should have skipped versions that already had a NeedsHumanReview flag (for the same reason) before - like `set_needs_human_review_on_latest_versions` does.